Snakefile

```diff
@@ -545,6 +548,11 @@ rule post_masked_consensus_and_summary_stats_to_id3c:
         if [ "$response" != "{params.id3c_expected_response_code}" ]
         then
             echo "Something went wrong in the ID3C POST request.\nSee {log} for more information."
+            curl -X POST -H 'Content-type: application/json' \
+                --data '{{"text": \
+                ":rotating_light: @{params.user} Assembly failed to upload to ID3C.\nMore details at `{log}`"}}' \
```

Comment:
Rather than constructing this JSON by hand, it'd be more robust to do something like:

```
params:
    slack_payload = {
        "text": f":rotating_light: @{params.user} Assembly failed to upload to ID3C.\nMore details at `{log}`"
    }
shell:
    … --data {json.dumps(params.slack_payload):q} …
```

Reply:
This is a good suggestion, but I am abandoning curl in favor of python requests.